combat-trainer.lic

```diff
@@ -978,7 +978,11 @@ class SpellProcess
 
     @settings = settings
 
-    @buff_spells = settings.buff_spells
+    if settings.combat_buff_waggle && settings.waggle_sets[settings.combat_buff_waggle]
```

Comment:
`combat_buff_waggle` needs added to `base.yaml` (and maybe `base-empty.yaml`)

Reply:
Added to `base.yaml` and expanded the existing comment around `buff_spells`. It didn't look like there were any other string style properties in `base-empty.yaml`, so I didn't put this one there either.